W-21352676 Integration Intelligence GA (#450)
mulesoft/docs-access-management: modules/ROOT/pages/_partials/include-integration-intelligence-setup.adoc

// tag::prereqs[]
Before enabling Integration Intelligence in your Anypoint Platform organization, you must:

> * Have an Anypoint Platform Integration Advanced or legacy Titanium package and Salesforce Unlimited edition.

Author: Unlimited, Enterprise

// tag::set-up-data-cloud[]

Connect your Salesforce organization to Data 360. Data 360 is the storage and processing layer for Integration Intelligence. It ingests MuleSoft telemetry data from your selected business group into a Data 360 data space.

. In the *Data Cloud* section, click *Setup*.
+
For information about setting up Data Cloud see https://help.salesforce.com/s/articleView?id=data.c360_a_set_up.htm&type=5[Set up and Maintain]. 

// end::set-up-data-cloud[]

// tag::enable-data-360-connectors[]
. Before enabling telemetry data, you must enable the Data 360 Connectors feature:
.. In Salesforce Setup, enter `Feature Manager` in the *Quick Find* box, then select *Feature Manager > Connectors (Beta)*.
.. Select *Enable*.

> .. Opt in to the beta terms by selecting the box, and then click *Enable*.

Reviewer: Do all the references to "beta" still apply here and elsewhere in this doc? I'm assuming yes. :)

Author: This one seems to still apply. This isn't part of Integration Intelligence--it's a Data 360 thing.

// end::enable-data-360-connectors[]

// tag::connect-with-anypoint-platform[]

Complete the connection between your Salesforce organization and Anypoint Platform so Salesforce can ingest MuleSoft telemetry data from your selected business group into Data 360.

. From *Setup*, in the Quick Find box, enter `MuleSoft`, and then select *MuleSoft > Anypoint Platform Setup*.
. On the *MuleSoft Anypoint Platform Setup* page, click *Complete the Connection*.
. Copy your Salesforce org tenant key. 
+
The format for the Salesforce tenant key is `cloud/env/org-id`.
. Sign in to Anypoint Platform using an account that has the root Organization Administrator permission.
. Select *Access Management* from the image::access-management::gear.svg["gear icon",20] menu.
. Select *Salesforce*.
. Click *Add Salesforce Org*.

> . In *Add the Salesforce Org*, paste the Salesforce org tenant key and click *Add*.

Reviewer: optional: add comma after "key" similar to following step. Up to you.

. Enter the name of the Salesforce org in *Name Used in Anypoint Platform*, and click *Add*.
+
Anypoint Platform generates and provides the organization key (or tenant key) in the format `cloud/env/org-id`.
. Click *Copy* to copy the Anypoint Platform organization key. 
. Return to the *MuleSoft Anypoint Platform Setup* page in the Salesforce organization.
. Enter the Anypoint Platform organization key in *Anypoint Platform Organization ID*, and then click *Connect*.
+ 
You return to the *MuleSoft Anypoint Platform Setup* page. The page shows the connected Anypoint Platform organization details.

// end::connect-with-anypoint-platform[]    

// tag::enable-telemetry-data[]

> Enable telemetry data so logs and metrics from your Mule apps and APIs are streamed into the selected Data 360 data space. 

Reviewer: optional
```suggestion
Enable telemetry data to stream logs and metrics from your Mule apps and APIs into the selected Data 360 data space. 
```

> . In the *Quick Find* box, enter `MuleSoft`, then select *MuleSoft Observability*. 

Reviewer: optional to follow pattern elsewhere:
, and then select

or 

, and select

. In *MuleSoft Observability*, switch *Telemetry Data Ingestion for Integration Intelligence* to *Active*. 
. Return to Anypoint Platform *Access Management > Salesforce*. 
. In *Capability enablement* for the connected Salesforce organization, switch *You can't bring telemetry data into Salesforce Data Cloud* to *Enabled*.

> . In *Enable Telemetry Data Ingestion*, select the business group that contains the apps that will send telemetry data to Data 360, then click *Next*.

Reviewer: optional:

, and then click

or 

, and click

Reviewer: Notebook seems good with ", and then"

+
[NOTE]
The business group you select for data ingestion can't be changed after you save the settings.
. In *Select Applications*, select the environment.
+ 

> A list of applications in the selected environment is shown.

Reviewer: ```suggestion
A list of applications in the selected environment appears.
```

per Notebook

. Select the applications to ingest telemetry data for, and click *Enable Data Ingestion*.
+
You can search for applications by name and you can also select *Show only applications available for selection* to filter the list and show only applications that use the latest runtime versions. 
+
The settings are saved.
+
Any subsequent changes made to Integration Intelligence in Access Management are automatically shared with the connected Salesforce organization. For example, turning off the Integration Intelligence feature in Anypoint Platform disables:

* Trust with the connected Salesforce organization. Turning it back on restores trust and enables data ingestion for the selected business group. 
* Seamless login for Anypoint Platform users to access Tableau Next dashboards in Salesforce. Turning it back on restores seamless login.

// end::enable-telemetry-data[]

// tag::enable-seamless-login[]

The Salesforce admin enables seamless login for Anypoint Platform users to access Tableau Next dashboards in Salesforce without needing to create new credentials or perform additional login steps.   

> . Log in to your Salesforce organization if you're not already logged in.

Reviewer: per Notebook, and says to avoid organization, but may be needed here?
```suggestion
. If you aren't logged in, log in to Salesforce.
```

[NOTE]
This step is available only after the Anypoint organization administrator completes all the Anypoint Platform setup steps.

. In Salesforce Setup, enter `MuleSoft` in the *Quick Find* box, then select *MuleSoft > Integration Intelligence Setup Assistant*.
. In *Install Integration Intelligence*, click *Setup*.
+

> The setup steps are shown. If a step isn't complete, click *Setup* to complete it. All the steps must be completed before you can install Integration Intelligence.

Reviewer: ```suggestion
The setup steps appear. If a step isn't complete, click *Setup* to complete it. All the steps must be completed before you can install Integration Intelligence.
```

. Select *Setup* for *Install Integration Intelligence*.
. In *Data Space*, select the data space to use for Integration Intelligence.
+
This is the data space where your observability data will be stored and processed and must match the data space where telemetry data is ingested.
. Click *Next*.

> . In *App Details*, enter a name and description for the Integration Intelligence app, then click *OK*.

Reviewer: optional: 

, and then
